test/func/fixtures/testplane/failed-describe.testplane.js

```diff
@@ -29,6 +29,10 @@ describe('failed describe', function() {
         throw new Error('Some error');
     });
 
+    it('failed test with ansi markup', async () => {
+        await expect({a: {b: 'c'}}).toMatchObject({c: {b: 'a'}});
```

Comment:
Do we need await here?

Reply:
actually no, but I added it for consistency